src/utils/package/buildPackageEnvs.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Builds the env object for the container
+ * 
+ * @function
+ * @param {object} params - Formatted arguments passed to the current task
+ * @param {object} contextEnv - context env for the container to run
+ * @returns {object}
+ */
+const buildPackageEnvs = ({ env, command, install }, contextEnv) => {
```

Comment:
Other tasks need to use this util. It's not just for the package tasks. It should be moved to the `utils/task` folder like we talked about. 
Also the name should be something like `convertParamsToEnvs`, or something like that